EpikCord/__init__.py

```diff
@@ -4186,4 +4188,90 @@ async def default_failure(self, interaction):
 class CommandUtils:
     
     def check(self, callback):
-        return Check(callback)
\ No newline at end of file
+        return Check(callback)
+
+class BaseConnectable:
+    def __init__(self,client:Client):
```

Comment:
Your code is not following the pep8 code style guidelines, which is a bit unfortunate since the repo as been formatted with black not long ago. I believe you should use it too, so we don't have to argue about code formatting.

```
pip install black
python -m black EpikCord
```

Reply:
And you shouldn't even be typehinting Client.

Reply:
1) I have implemented this black formatting. Thank You 
2) I have typehinted client just for reference